ConsoleFileManager/Program.cs

```diff
@@ -0,0 +1,5 @@
+﻿using ConsoleFileManager;
+using ConsoleFileManager.Services;
+using FileManager.Services;
+
+new ConsoleFileManagerLogic(OSNavigator.Navigator, new ConsoleMessageService()).Start();
```

Comment:
```suggestion
var logic = new ConsoleFileManagerLogic(OSNavigator.Navigator, new ConsoleMessageService());
logic.Start();
```

Пишите корректно. Сперва создать объект, положить в переменную. Потому вызвать у него метод. Код должен быть читаемым.